gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -135,8 +144,11 @@ typedef struct _GstTensorFilterProperties
   int latency; /**< The average latency over the recent 10 inferences in microseconds */
   int throughput; /**< The average throughput in the number of outputs per second */
   int invoke_dynamic; /**< True for supporting invoke with flexible output. */
-
+  int invoke_async; /**< Use this when the sub-plugin receives an input and generates multiple outputs asynchronously. */
```

Comment:
We are now breaking ABI compatibility by updating GstTensorFilterProperties.

To minimize future impact for further updates, let's add reserved bytes (around 100~200 bytes?) and add the new properties after previous properties.

Reply:
Okay, I will update.

Reply:
I've updated it. Thank you.